modules/ROOT/pages/setup.adoc

```diff
@@ -79,8 +83,12 @@ wrapper.java.maxmemory=4606
 
 ----
 
+. Install your license. See https://docs.mulesoft.com/mule-runtime/4.2/installing-an-enterprise-license[Install an Enterprise License] for more information.
+
 . Add the server in Runtime Manager based on the Runtime manager https://docs.mulesoft.com/runtime-manager/servers-create#create-a-server[Create a Server].
 
+. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
+
 . Include the created servers into the server group you created earlier: *B2BServerGroup*: https://docs.mulesoft.com/runtime-manager/server-group-add[Add Servers to a Group]
 
 . Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
```

Comment:
```suggestion
. Start the Mule runtime and verify it is active within Anypoint Runtime Manager.
```
Start the Mule runtime and verify it is shown as Running in Anypoint Runtime Manager.

Reply:
Fixed.